packages/blocks/slack/src/lib/actions/request-action-message.ts

```diff
@@ -106,6 +104,27 @@ const sendMessageAskingForAction = async (
     conversationId,
   );
 
+  const disableSlackInteractions =
+    system.getBoolean(SharedSystemProp.DISABLE_SLACK_INTERACTIONS) || false;
```

Comment:
for those that already have an old installation, they won't have it in their .env file, so by default I'm keeping the old behavior

Reply:
A very confusing variable name.

We now have two slack actions mode - interactions and URL. So we can just call this variable SLACK_ACTIONS_MODE which can be either URL or interactions.

If it must stay as a boolean, it should be `ENABLE_SLACK_INTERACTION_MODE` or `ENABLE_SLACK_ACTIONS_BY_URL`, definitely not a "DISABLE" variable.